docker/main/dataservice/Dockerfile

```diff
@@ -8,11 +8,22 @@ ARG service_package_name
 ARG service_package_constraint
 WORKDIR /code
 COPY ./entrypoint.sh entrypoint.sh
+# Copy custom built packages from external sources image
 COPY --from=sources /DIST /DIST
 RUN apk update \
-    && apk add --no-cache openssl git g++ libffi-dev \
+    && apk add --no-cache openssl git g++ libffi-dev proj proj-dev proj-util gdal-dev openblas openblas-dev \
     && chmod u+x entrypoint.sh \
     && pip install --no-cache-dir --upgrade pip \
-    && pip install --no-cache-dir --find-links=/DIST "${service_package_name}${service_package_constraint}" \
+    && pip install --no-cache-dir aiohttp==3.8.1 \
+    && pip install --no-cache-dir --no-index --find-links=/DIST "${service_package_name}${service_package_constraint}" \
+    # TODO: this is a workaround, and may have side effects (transitive dependency issue with aiohttp and redis) \
+    && pip install --no-cache-dir aiohttp==3.8.1 \
```

Comment:
I can't remember exactly at this point, but I think so.  This may be three times, actually, with the first being redundant, because of transitive dependency constraints when installing the service package.